### PR TITLE
Dev design

### PR DESCRIPTION
**Pull request by FarazA22**
### Ticket

https://www.pivotaltracker.com/story/show/184606287

### Approach
Add Action Bar Story Book Component and Update CTA section across the app

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/components/tasks/business-formation/DbaFormationPaginator.tsx [hunk 1]
```diff
@@ -151,48 +150,72 @@ export const DbaFormationPaginator = (): ReactElement => {
...
-          text={state.dbaContent.Authorize.callToActionText}
-          onClick={(): void => setShowCtaModal(true)}
-        >
-          <FormationHelpButton />
-          <BackButton />
-        </TaskCTA>
+        <>
+          <CtaContainer>
+            <ActionBarLayout>
+              <>
```
Comment (seidior) [inline web/src/components/tasks/business-formation/DbaFormationPaginator.tsx:188]: [dust] Do we still need this Fragment?

File: web/src/components/tasks/business-formation/NexusFormationFlow.tsx [hunk 2]
```diff
@@ -83,18 +85,38 @@ export const NexusFormationFlow = (): ReactElement => {
...
-                isNotFullWidthOnMobile={true}
-                onClick={onPreviousButtonClick}
-              >
-                {Config.formation.general.previousButtonText}
-              </SecondaryButton>
-            </TaskCTA>
+
+            <CtaContainer>
+              <ActionBarLayout>
+                <>
```
Comment (seidior) [inline web/src/components/tasks/business-formation/NexusFormationFlow.tsx:91]: [dust] Likewise, this Fragment.

File: web/src/pages/onboarding.tsx [hunk 3]
```diff
@@ -474,12 +474,12 @@ const OnboardingPage = (props: Props): ReactElement => {
                           data-testid={`page-${index + 1}-form`}
                         >
                           {onboardingPage.component}
-                          <hr className="margin-top-6 margin-bottom-4" aria-hidden={true} />
-                          <DevOnlySkipOnboardingButton setPage={setPage} routeToPage={routeToPage} />
+                          <hr className="margin-top-6 margin-bottom-2" aria-hidden={true} />
```
Comment (seidior) [inline web/src/pages/onboarding.tsx:477]: [dust] Any particular reason we're `aria-hidden`-ing a presentational element?

File: web/src/pages/profile.tsx [hunk 4]
```diff
@@ -769,27 +773,35 @@ const ProfilePage = (props: Props): ReactElement => {
...
-                                isRightMarginRemoved={true}
-                                dataTestId="save"
-                                isLoading={isLoading}
-                              >
-                                {Config.profileDefaults.default.saveButtonText}
-                              </PrimaryButton>
-                            </div>
+                            <ActionBarLayout>
+                              <>
+                                <div className={isXSMobile ? "margin-top-2" : "display-inline"}>
```
Comment (seidior) [inline web/src/pages/profile.tsx:778]: [pebble] Can this be accomplished with:

```typescript
<div className="margin-top-2 display-block mobile-lg:margin-top-0 mobile-lg:display-inline">
```

File: web/src/pages/profile.tsx [hunk 5]
```diff
@@ -769,27 +773,35 @@ const ProfilePage = (props: Props): ReactElement => {
...
+                                  <SecondaryButton
+                                    isColor="primary"
+                                    onClick={(): Promise<void> => onBack()}
+                                    dataTestId="back"
+                                  >
+                                    {Config.profileDefaults.default.backButtonText}
+                                  </SecondaryButton>
+                                </div>
+                                <div>
+                                  <div className={isXSMobile ? "" : "display-inline"}>
```
Comment (seidior) [inline web/src/pages/profile.tsx:788]: [pebble] Can this be accomplished with:

```typescript
<div className="mobile-lg:display-inline">
```

File: web/stories/Molecules/action-bar/DefaultActionBar.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+const Template = () => {
+  const isXSMobile = useMediaQuery(MediaQueries.isXSMobile);
+
+  return (
+    <div className="tablet:width-tablet">
+      <hr className="margin-bottom-2" />
+      <ActionBarLayout>
+        <>
+          <div className={isXSMobile ? "margin-top-2" : ""}>
```
Comment (seidior) [inline web/stories/Molecules/action-bar/DefaultActionBar.stories.tsx:16]: [pebble] Likewise from previous comments.

File: web/stories/Molecules/action-bar/DefaultActionBar.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+            <SecondaryButton isColor="primary" onClick={() => {}} dataTestId="back">
+              SecondaryButton
+            </SecondaryButton>
+          </div>
+
+          <div className={isXSMobile ? "" : "display-inline"}>
```
Comment (seidior) [inline web/stories/Molecules/action-bar/DefaultActionBar.stories.tsx:22]: [pebble] Likewise from previous comments.

**Review by seidior — COMMENTED** [review]
Thank you so very much for camp grounding my `<TaskFooter />` and related components! The work looks great here. I have a few comments about your work, some of which are just questions that can be answered.

File: web/src/components/njwds-extended/cta/SingleCtaLink.tsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+            onClick={(): void =>
+              analytics.event.task_primary_call_to_action.click.open_external_website(
+                text || Config.taskDefaults.defaultCallToActionText,
+                link as string
+              )
+            }
+          >
+            {text || Config.taskDefaults.defaultCallToActionText}
+          </PrimaryButton>
+        </a>
```
[the remark this answers is not in the document]
Comment (AnJuHyppolite) [reply web/src/components/njwds-extended/cta/SingleCtaLink.tsx:33]: Curious about this too.

**Pull request merged**